Fachportal Padagogik.js

```diff
@@ -102,31 +104,15 @@ function scrape(doc, url) {
 		});
 	} else {
 		var pdfUrl = ZU.xpathText(doc, '//meta[@name="citation_pdf_url"]/@content');
-		Z.debug(pdfUrl);
-
-		/*
-		//fails on http://www.pedocs.de/frontdoor.php?source_opus=9491&la=de
-		Z.debug(ZU.xpathText(doc, '//span[@class="Z3988"]/@title'));
-		var translator = Zotero.loadTranslator("web");
-		translator.setTranslator("05d07af9-105a-4572-99f6-a8e231c0daef");//COinS translator
-		translator.setDocument(doc);//TODO encoding is normally Western and not utf8
-		translator.setHandler("itemDone", function (obj, item) {
-			if (pdfUrl) {
-				item.attachments.push({
-					url: pdfUrl,
-					title: "Full Text PDF",
-				});
-			}
-			
-			finalize(doc, item);
-		});
-		translator.translate();
-		*/
+		//Z.debug(pdfUrl);
 		
 		var coins = ZU.xpathText(doc, '//span[@class="Z3988"]/@title');
 		if (coins) {
+			coins = coins.replace(/%26%238208%3B/g, '-')
+				.replace(/%C2%82/g, '‚').replace(/%C2%91/g, '‘');
```

Comment:
This works nicely! Thank you! I included it in the new version. Please have a look.